docs/source/release/v4.2.0/framework.rst

```diff
@@ -14,7 +14,8 @@ Concepts
 
 Algorithms
 ----------
-* ref:`MaskAngle <algm-MaskAngle>` has an additional option of ``Angle='InPlane'``
+* :ref:`MaskAngle <algm-MaskAngle>` has an additional option of ``Angle='InPlane'``
+* Whitespace is now ignored anywhere in the string when setting the Filename parameter in :ref:`Load`.
```

Comment:
So rather unhelpfully GitHub as rendered my previous comment inside the preformat ticks not as I would have expected it. *:ref:`Load`* should be *:ref:`Load <algm-Load>`* and that will solve the ubuntu documentation warning. Sorry about that.